.github/workflows/build.yml

```diff
@@ -47,6 +47,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Setup redis
+        uses: supercharge/redis-github-action@1.4.0
+        with: 
+          redis-version: 6
+
+
```

Comment:
One empty line too much here? ;-)

I guess the [rest of the implementation](https://stackoverflow.com/questions/67619624/do-i-need-docker-to-run-redis-on-github-actions/67631989#67631989) would only test that the server is up, and we probably don't need it each time we run the build, right?

```
      - name: Install redis cli # so we can test the server
        run: sudo apt-get install -y redis-tools

      - name: Verify that redis is up
        run: redis-cli ping
```

Great, thanks :+1:

Reply:
Yep, I was thinking the same ... ;)